aten/src/ATen/native/Normalization.cpp

```diff
@@ -593,7 +593,8 @@ Tensor group_norm(const Tensor& input, int64_t num_groups,
              " and input of shape ", input.sizes());
 
     // Apply group norm
-    auto input_reshaped = input.contiguous().view({1, b * num_groups, -1});
+    // view(..., -1) does not work for empty tensor
+    auto input_reshaped = input.contiguous().view({1, b * num_groups, b ? -1 : 1});
```

Comment:
should we also file an issue about `view` not supporting -1 for empty tensors?

Reply:
It is deliberately not supported due to ambiguity.

Reply:
TensorFlow does support reshape empty tensors using -1, by just ignoring the empty dims. It actually works quite well in practice (though not so intuitively when facing wrong inputs). For example this current line I'm changing would work out of the box.